llvm/test/CodeGen/DirectX/ShaderFlags/disable-opt-cs.ll

```diff
@@ -8,7 +8,7 @@
 ; CHECK-NEXT: ;       Disable shader optimizations
 
 ; CHECK: ; Shader Flags for Module Functions
-; CHECK: ; Function main : 0x00000000
+; CHECK: ; Function main : 0x00000001
```

Comment:
Do these disable-opt-*.ll tests make sense anymore? Now that that block of code is removed?

Reply:
The Disable Optimizations flag still exists, so it does still need to be tested.
In this case, the flag is now set on each function instead of only for the whole module, so the tests have been modified to reflect that.